eng/pipelines/runtime.yml

```diff
@@ -410,6 +410,7 @@ jobs:
     platforms:
     - Android_x86
     - Android_arm64
+    - Linux_arm
```

Comment:
@akoeplinger do you think we need Linux_arm debug on the libraries build?

Reply:
No, the existing build should be enough just like we do for Linux_arm64 today: https://github.com/dotnet/runtime/blob/f007634b02c4484f60ead75a893eb7f89f615ccf/eng/pipelines/runtime.yml#L390

Reply:
Ok, switching it to use the release build of libraries.